src/assets.rs

```diff
@@ -191,42 +191,50 @@ impl HighlightingAssets {
         input: &mut OpenedInput,
         mapping: &SyntaxMapping,
     ) -> &SyntaxReference {
-        let syntax = if let Some(language) = language {
+        let syntax = if match input.kind {
+            OpenedInputKind::ThemePreviewFile => true,
+            _ => false,
+        } {
+            // FIXME: replace the above match statement with matches macro when min Rust >= 1.42.0
```

Comment:
:+1: 

In the meantime, could we move this logic to a helper function in the `impl OpenedInputKind` block? This would be much easier to read:

```
let syntax = if input.kind.is_theme_preview_file() {
```